src/devtools.ts

```diff
@@ -58,8 +59,8 @@ export function preactDevtoolsPlugin({
 		transform(code, url) {
 			const { id } = parseId(url);
 
-			if (entry === id && (!config.isProduction || injectInProd)) {
-				const source = injectInProd ? "preact/devtools" : "preact/debug";
+			if (entry === id) {
+				const source = devToolsEnabled ? "preact/devtools" : "preact/debug";
```

Comment:
I'm a bit confused by this, or how it's meant to be working. Why would the source change based upon whether `injectInProd` is true vs false? As `injectInProd` is `false` by default, even in dev, this would return `'preact/debug'`, which seems wrong?

Just curious if I'm missing something, fair enough if this was a bug from changes elsewhere. Just couldn't find much context in the blame.